src/pages/ProjectSetup.test.jsx

```diff
@@ -38,3 +41,35 @@ test("renders page", () => {
   const label9 = screen.getByText("High");
   expect(label9).toBeInTheDocument();
 });
+
+test("Testing inputs work makes post call", async () => {
+  const pageData = {
+    title: "Test Project",
+    acronym: "TP",
+    id: 1,
+  };
+  let mock = new MockAdapter(axios);
+  mock.onPost(`${config.backendUrl}/projects/`).reply(200, pageData);
+  render(<ProjectSetup />, { wrapper: MemoryRouter });
+  fireEvent.click(screen.getByRole("button", { name: "Next" }));
+  fireEvent.change(screen.getByLabelText("Full name"), {
+    target: { value: "Full Name" },
+  });
+  fireEvent.change(screen.getByLabelText("Acronym"), {
+    target: { value: "Acronym" },
+  });
+  fireEvent.click(screen.getByLabelText("CMS AWS Commercial East-West"));
+  fireEvent.click(screen.getByRole("button", { name: "Next" }));
```

Comment:
The reason I keep calling click next is to validate the individual checks that I had to put on each radio button in order for code coverage to increase...